.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ on:
   push:
     branches: [ develop ]
   pull_request:
-    branches: [ develop ]
+    branches: [ develop, master ]
```

Comment:
For what?
In other services it is just develop

Reply:
I made it like ProjectService